odtp/cli/execution.py

```diff
@@ -41,6 +44,23 @@ def prepare(
             document_id=execution_id, 
             collection=db.collection_executions
         )
+
+        if not project_path:
+            #Inferring project path from naming convention
```

Comment:
Can you please put this also into a helper: how to get the project-path for an execution from the execution_id: that way I could also consider to use it on the gui.

That function can then also be reused by `execution run`